packages/schema-dot-org-types/src/generated/alignment-object.js

```diff
@@ -0,0 +1,30 @@
+//@flow
```

Comment:
There's still manual steps–you have to invoke schema-transform on the types you care about. I'm not excited to try to automate that and deal with error recovery, etc.